### PR TITLE
feat(apigateway): default value for enum type in schema models

### PR DESCRIPTION
**Pull request by mmuller88**
Fixes: #11065
Add default to JsonSchema

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Comment by mmuller88** [comment]
defaultProp

**Comment by mmuller88** [comment]
I assume the unit test is failing. Can someone help me to understand how to run those pls ?

**Comment by mmuller88** [comment]
@nija-at jaii I got the unit test passing :)

File: packages/@aws-cdk/aws-apigateway/package.json [hunk 1]
```diff
@@ -244,6 +244,8 @@
       "docs-public-apis:@aws-cdk/aws-apigateway.JsonSchema.contains",
       "props-default-doc:@aws-cdk/aws-apigateway.JsonSchema.contains",
       "docs-public-apis:@aws-cdk/aws-apigateway.MethodProps",
+      "docs-public-apis:@aws-cdk/aws-apigateway.JsonSchema.default",
+      "props-default-doc:@aws-cdk/aws-apigateway.JsonSchema.default",
```
Comment (nija-at) [inline packages/@aws-cdk/aws-apigateway/package.json:248]: These shouldn't be required.

**Review by nija-at — CHANGES_REQUESTED** [review]
Thanks for submitting this PR.

File: packages/@aws-cdk/aws-apigateway/package.json (hunk 1 above)
Comment (mmuller88) [reply packages/@aws-cdk/aws-apigateway/package.json:248]: thx @nija-at I thought the same but the build was failing without those.

Comment (mmuller88) [reply packages/@aws-cdk/aws-apigateway/package.json:248]: ah just the default on you mean

Comment (nija-at) [reply packages/@aws-cdk/aws-apigateway/package.json]: Neither should be required. One is stating that the property has no documentation and the other is stating that the default is not documented. You have both specified.

**Review by nija-at — CHANGES_REQUESTED** [review]
Please see my response at https://github.com/aws/aws-cdk/pull/11064#discussion_r515173985

File: packages/@aws-cdk/aws-apigateway/package.json (hunk 1 above)
Comment (mmuller88) [reply packages/@aws-cdk/aws-apigateway/package.json]: Than the build should fail. I show you.

Comment (mmuller88) [reply packages/@aws-cdk/aws-apigateway/package.json]: Hi @nija-at . Indeed it passed. I am sure it failed before without that two lines. Anyway. Thank you for baring with me :). Cheers

**Review by nija-at — APPROVED** [review]

**Pull request merged**